README.adoc

```diff
@@ -1,41 +1,116 @@
 == Quick development start
 
-To build and run from source:
+The project is ready to start the application in
+https://tbroyer.github.io/gwt-maven-plugin/codeserver-mojo.html[GWT SuperDevMode].
+In SuperDevMode, the client side of the application is served by GWT's CodeServer, which can quickly
+recompile changes made in the source files in your IDE and serve the updated client upon browser refresh.
+On the other hand, the webapp is deployed to an AppServer running in a separate process and only provides
+the server side of the application.
+This way it is possible to reduce the refresh time during development either by just refreshing browser window
+to get client code changes or by just redeploying the webapp (without recompiling the client) to see backend changes.
 
-----
-$ mvn clean install -DskipTests
+Additionally, the GWT Maven plugin is configured to produce source maps which allow to browse the Java source files,
+set breakpoints and inspect variables using the Developer tools in Chrome.
+
+=== Running SuperDevMode from command line
+
+==== 1. Build the whole project
+
+```bash
+$ mvn clean install -Dgwt.skipCompilation
+```
+
+We can skip GWT compilation because it will be done anyway by the CodeServer
+when accessing the application from browser for the first time.
+
+==== 2. Launch CodeServer
```

Comment:
Yes. It is necessary when starting SDM from command line so that CodeServer first injects a `*.nocache.js` file with a URL pointing to the CodeServer to the webapp before it's deployed. This is configured using the CodeServer's `launcherDir` argument.

When running SDM in IDEA, the webapp is deployed in a different way so that any changes in target/webapp are reflected in the running app. So it works even if the CodeServer is started after the webapp.